itest/loadtest/mint_batch_test.go

```diff
@@ -127,7 +127,12 @@ func mintTest(t *testing.T, ctx context.Context, cfg *Config) {
 	// we asserted previously.
 	uniRoots, err := alice.AssetRoots(ctx, &unirpc.AssetRootRequest{})
 	require.NoError(t, err)
-	require.Len(t, uniRoots.UniverseRoots, groupCount)
+	issuanceRoots := fn.FilterMap(
+		uniRoots.UniverseRoots, func(root *unirpc.UniverseRoot) bool {
```

Comment:
Ahh I think this is what you were running into @calvinrzachman ?

Reply:
Yes, exactly.

Reply:
Yea, deployed to `staging` and this change seems to have fixed the issue! 🚀